infra/resources/modules/commons/containers.tf

```diff
@@ -83,6 +84,56 @@ resource "azurerm_cosmosdb_sql_container" "activations" {
   }
 }
 
+resource "azurerm_cosmosdb_sql_container" "activation" {
+  name                  = "activation"
+  resource_group_name   = azurerm_resource_group.data_rg.name
+  account_name          = module.cosmosdb_account.name
+  database_name         = module.cosmosdb_sql_database_trial.name
+  partition_key_path    = "/trialId"
+  partition_key_version = 2
+
+  # A unique key is scoped to a logical partition.
+  # Remember also that sparse unique keys are not supported. If some unique path
+  # values are missing, they're treated as null values, which take part in the
+  # uniqueness constraint. For this reason, there can be only a single item with
+  # a null value to satisfy this constraint.
+  unique_key {
+    paths = ["/userId"]
+  }
+
+  autoscale_settings {
+    max_throughput = 10000
```

Comment:
**note**: IMHO this can be lower, but for now, let's keep the same value as `activations`.